service-base/src/main/java/org/eclipse/hono/service/amqp/AmqpServiceBase.java

```diff
@@ -385,6 +384,79 @@ protected final ResourceIdentifier getResourceIdentifier(final String address) {
         }
     }
 
+    /**
+     * Handles a request from a client to establish a link for sending messages to this server.
+     * The already established connection must have an authenticated user as principal for doing the authorization check.
+     *
+     * @param con the connection to the client.
+     * @param receiver the receiver created for the link.
+     */
+    protected void handleAuthorizedReceiverOpen(final ProtonConnection con, final ProtonReceiver receiver) {
```

Comment:
This is a good proposal - thought the tests depend more on HonoMessaging and thus would be hard to move.
Turned out to work perfectly, I will introduce a new test class then. Thanks for the review :-)